WeakAurasOptions/Locales/enUS.lua

```diff
@@ -65,7 +67,8 @@ L["Border Size"] = "Border Size"
 L["Border Style"] = "Border Style"
 L["Bottom Text"] = "Bottom Text"
 L["Button Glow"] = "Button Glow"
-L["Can be a name or a UID (e.g., party1). A name only works on friendly players in your group."] = "Can be a name or a UID (e.g., party1). A name only works on friendly players in your group."
+L["Can be a name or a UID (e.g., party1). A name only works on friendly players in your group."] =
```

Comment:
add this file to `.styluaignore`?

Reply:
should be fine with it just being on the next line, still valid Lua